static/js/zamboni/reviewers.js

```diff
@@ -223,7 +223,7 @@ function initDailyMessage(doc) {
     $motd.find('.close').show();
     if (storage.get('motd_closed') != $('p', $motd).text()) {
         // You haven't read this spam yet? Here, I have something to show you.
-        $motd.slideDown();
+        $motd.show();
```

Comment:
No slide? Sad!

Reply:
I was tired of seeing the page reflow every time I was loading it... and I figured I wasn't alone :)